spec/logger.spec.js

```diff
@@ -0,0 +1,57 @@
+describe("logger", function () {
+    "use strict";
+
+    var log, logger;
+
+    logger = require("../lib/logger");
```

Comment:
Have to leave variables out of before each so I can use them in tests, but the require I moved into beforeEach.